.travis/install.sh

```diff
@@ -37,5 +37,5 @@ else
   popd
 fi
 
-cd pulp_python
```

Comment:
This should be incorrect, after entering the directories above it does a popd, so it should be ending up where it started.

Is that not happening properly?

Reply:
It is not: https://travis-ci.org/pulp/pulp_python/jobs/386389531

Reply:
ohhh I see this is after we renamed the branch so the branch checkout errors out and it doesn't popd

Reply:
@werwty this is why

https://travis-ci.org/pulp/pulp_python/jobs/386389531#L550

The branch 3.0-dev no longer exists, therefore the ```git checkout``` errors, therefore bash's ```&&``` expression falls through early.

Reply:
@werwty it does still need the ```cd pulp_python``` line though

Also could you edit the install docs?